tests/fixtures/Item.php

```diff
@@ -7,11 +7,11 @@ class Item implements Comparable {
 	
 	private $content;
 	
-	public function __construct($content = '') {
+	public function __construct(string $content = '') {
 		$this->content = $content;
 	}
 	
-	public function compareTo($comparison) {
+	public function compareTo($comparison): int {
```

Comment:
I think it should add type hint for `$comparison`.

Reply:
Disagree, because the signature should be compatible with `phootwork\lang\Comparable` interface (line 15). Adding type hint here translates in a fatal error.

Reply:
Hi @cristianoc72, thank you for your explanation.

That's right. To be compatible with `phootwork\lang\Comparable` interface, we don't have to add type hint for this variable.